ci/jest/env.js

```diff
@@ -1,13 +1,26 @@
 const tracer = require('../../packages/dd-trace')
 const { ORIGIN_KEY } = require('../../packages/dd-trace/src/constants')
 
-tracer.init({
+const options = {
```

Comment:
Any way to reuse the options between the entrypoints?

Reply:
yeah this is a bit awkward because of the custom `flushInterval` for `jest`. I'll try to come up with something